_compdemos/intro-github.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Publishing code and data on GitHub
+last_modified_at: 2021-02-09
+main_author: Lauren Wolfe
+primary_reviewers: k8hertweck
+---
+
+This tutorial is meant to act as a step-by-step guide for researchers interested in publishing code and data associated with a manuscript. While there are different ways to go about this task, this document is focused specifically on publishing to GitHub.
+
+Using GitHub requires and basic understanding of:
```

Comment:
Most of this material is already outlined in the [Managing and Sharing Code](https://sciwiki.fredhutch.org/scicomputing/software_managecode/) wiki page, as well as the [Intro to GitHub](https://fredhutchio.github.io/git_github_intro/) course. It would be worth checking out the existing wiki page and seeing if anything is missing (there are a few issues already filed about some details)>